#if !defined(_BASE_INCLUDE_H_)
#define _BASE_INCLUDE_H_

#include "log_driver/log_driver.h"
#include "utils/CJsonObject.hpp"
#include "utils/Base64.h"
#include "utils/clipp.h"
#include "db_orm/zh_db_config.h"
#include "idl/gen_code/cpp/config_management.h"
#include "idl/gen_code/cpp/idl_types.h"
#include "idl/gen_code/cpp/rbac_center.h"
#include "idl/gen_code/cpp/device_management.h"
#include "idl/gen_code/cpp/order_center.h"
#include "utils/utils.h"
#include "utils/tdf_timer.h"
#include "log_driver/cli.h"
#include "file_up_down/file_up_down.h"
#include "serial_link/zh_vcom_link.h"
#include "zyzl_plugin/zh_zyzl.h"
#include "log_driver/clifilesession.h"
#include "log_driver/clilocalsession.h"
#include "log_driver/loopscheduler.h"


#endif // _BASE_INCLUDE_H_

